xml/art-obs-beginners-guide.xml

```diff
@@ -175,24 +167,28 @@ https://www.suse.com/communities/blog/suse-studio-integration/
    <title>Projects and Packages</title>
    <para>
     In &obsa;, packages are organized in <emphasis>projects</emphasis>.
-    A single project can contain several packages and serves a specific
-    purpose, defines access control, other repositories, and build targets
-    (operating systems and architectures).
+    A single project can contain several packages and usually serves a specific
+    organizational purpose. Generic access control, related repositories, and
+    build targets (operating systems and architectures) are all defined at
+    the project level.
    </para>
    <para>
     Projects can also contain other projects (subprojects) to structure work.
     Each project name is separated by colons.
-    For example, all packages for fonts are collected in the project
-    <uri>M17N:fonts</uri> and all relevant packages for the Python programming
-    language are available in the <uri>devel:languages:python</uri> project.
+    For example, in the &opensuse; Build Service, packages for fonts are
+    collected in the project
+    <uri>M17N:fonts</uri> which is a subproject of <uri>M17N</uri>.
+    Packages for the Python programming
+    language are available in the <uri>devel:languages:python</uri> project
+    which is a subproject of <uri>devel:languages</uri> which itself is a
+    subproject of <uri>devel</uri>.
    </para>
    <para>
-    As a user, you will build your packages normally in your
+    As a user, you will normally build packages in your
     <firstterm baseform="Home Project">home project</firstterm>, available
     in &obsa; as <uri>home:<replaceable>USERNAME</replaceable></uri>.
-    The home project is your working area in &obsa; for defining build targets,
-    uploading and building packages, and installing the results with Zypper
-    or &yast;.
+    Home projects serve as a personal working area in &obsa; to define build
+    targets, upload, build, and download packages.
```

Comment:
hm, long sentence .... maybe stop behind the "download packages" and continue with "Users are also permitted further sub projects for isolated work area". 

The branches are just one example where this is used in the end.